src/living/cssom/parsers/index.ts

```diff
@@ -950,3 +951,87 @@ export function shorthandSetter(
     }
   };
 }
+
+function processArgs(
+  args: string[], 
+  converter: (arg: string) => PropertyLengthValue | PropertyAngleValue
+): PropertyLengthValue[] | PropertyAngleValue[] {
+  let values = [];
+  for (let arg of args) {
+    const value = converter(arg);
+    if (value === undefined) {
+      values.length = 0;
+      break;
+    } else {
+      values.push(value);
+    }
+  }
+  return values;
+}
+
+function createAndAddTransformFunction(
+  constructor: new (name: string, args: string[]) => UnionTransformFunction,
+  name: string,
+  args: string[],
+  parsedTransformFunctions: UnionTransformFunction[]
+): boolean {
+  const transformFunction = new constructor(name, args);
+  if (transformFunction.isValid) {
+    parsedTransformFunctions.push(transformFunction);
+    return true;
+  }
+  return false;
+}
+
+export class TransformFunction<Tv> {
+  name: string;
+  values: Array<Tv>;
+
+  constructor(name: string, values: Tv[] = []) {
+    this.name = name;
+    this.values = values;
+  }
+}
+
+export class TranslationTransformFunction extends TransformFunction<PropertyLengthValue> {
+  constructor(name: string, args: string[]) {
+    const values = processArgs(args, toLengthStr) as PropertyLengthValue[];
+    super(name, values);
+  }
+  get isValid(): boolean {
+    return this.values.length > 0;
+  }
+}
+
+export class RotationTransformFunction extends TransformFunction<PropertyAngleValue> {
+  constructor(name: string, args: string[]) {
+    const values = processArgs(args, toAngleStr) as PropertyAngleValue[];
+    super(name, values);
+  }
+  get isValid(): boolean {
+    return this.values.length > 0;
```

Comment:
```suggestion
    return this.values.length === 1;
```

See: https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate